src/scss/global.scss

```diff
@@ -133,6 +133,11 @@ span.cm-def {
   background: $color-editor !important;
 }
 
+.CodeMirror-dialog-bottom {
+    position: fixed; 
+    bottom:0%;
```

Comment:
looks like you've got some whitespace problems here: could you maybe also put a comment as to why this rule is necessary for the keymap functionality?